### ISSUE TITLE
twisted.python.util.LineLog needs to be visible on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8790]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8790|
|Type|enhancement|
|Created|2016-08-27 16:07:54Z|

The class twisted.python.util.LineLog is not available on Python 3.
It needs to be made available so that modules such as twisted.mail can be
ported to Python 3.

<details><summary>Searchable metadata</summary>

```
trac-id__8790 8790
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1472314074230794 1472314074230794
changetime__1472326041639963 1472326041639963
version__None None
owner__Craig_Rodrigues__rodrigc_____ Craig Rodrigues <rodrigc@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/529

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_2"></a>|@acabhishek942 commented|
|-|-|

changes looks good. Please merge

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc set owner to @rodrigc<br>@rodrigc set status to `closed`|
|-|-|

In changeset bf6c8661a51d7fe4fa5340a4f1f9d51e1e1d52ac
```
#!CommitTicketReference repository="" revision="bf6c8661a51d7fe4fa5340a4f1f9d51e1e1d52ac"
Merge pull request #529 from twisted/8790-rodrigc-linelog

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8790

twisted.python.util.LineLog needs to be visible on Python 3
```
